Services/Language/classes/Setup/class.ilSetupLanguage.php

```diff
@@ -567,7 +567,7 @@ protected function insertLanguage($lang_key, $scope = '')
                 }
             }
     
-            $query = "INSERT INTO lng_modules (module, lang_key, lang_array) VALUES ";
+            $query = "REPLACE INTO lng_modules (module, lang_key, lang_array) VALUES ";
```

Comment:
@PurHur Wouldn't this mean that entries that are no longer in the file are never deleted from the database?

in the original code, isn't the sequence of INSERT and subsequent DELETE simply wrong?